app-modules/audit/src/Overrides/Concerns/AttachOverrides.php

```diff
@@ -43,6 +43,11 @@
 
 trait AttachOverrides
 {
+    /**
+     * @param array<string, mixed> $attributes
+    * @param mixed $id
+    * @param mixed $touch
```

Comment:
These should probably be in order and the weird spacing on the first one fixed.

Also it looks like `$touch` is a boolean, not mixed.